src/oop/user_defined_classes/medium/q6/QuestionText.md

```diff
@@ -0,0 +1,22 @@
+# oop-user_defined_classes-medium-q6
+
+## Java
+
+Teachers need to have an efficient way to keep track of each student and how they do in 
+their course work. Make a Java class called `Homework` that has two attributes called **marks**
+and **studentNumber** that are initialized by the constructor.
+
+
+
+
+## Homework
+
+The `Homework` class you are given has the following items:
+
+- Constructors:
+    - A constructor that initializes the **marks** and **studentNumber** attributes
+
+You need to add the following items:
+- Attributes:
+    - Write the required attributes according to the initialization and parameters of
```

Comment:
Please change the description to specify the names, types, and visibility modifier